tests/core/pyspec/eth2spec/test/helpers/phase1/attestations.py

```diff
@@ -1,11 +1,13 @@
+# TODO: What is this file for??? It seems to be broken!
```

Comment:
@djrtwo @protolambda Couldn't really see the reason for this file

Reply:
Phase 1 required some specific things of attestations that had no place in phase0, so I think this was added to fill that gap. The phase0 attestation helper file still has a few special cases for phase1 attestations though, it would be nice to separate that, while avoiding code duplication.

Reply:
fyi I removed `helpers/phase1/attestations.py` and added phase 1 specific logic in `test/helpers/attestations.py ` in #1703 too.